src/ДЗ. TypeScript. Примітивні літеральні типи та типізація обʼєктів.ts

```diff
@@ -1,188 +1,168 @@
-<<<<<<< HEAD
 class School {
   // implement 'add area', 'remove area', 'add lecturer', and 'remove lecturer' methods
 
-  _areas = [];
-  _lecturers = []; // Name, surname, position, company, experience, courses, contacts
+  _areas: string[] = [];
+  _lecturers: (string | number)[] = []; // Name, surname, position, company, experience, courses, contacts
```

Comment:
Please, provide object type annotation with fields that mentioned in comment instead of union (string | number) type.

Reply:
Yes, that's correct. That's exactly what I meant. :-)